build_tools/cmake/run_riscv64_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2020 The IREE Authors
```

Comment:
nit: script looks new, could use 2022 for the year
```suggestion
# Copyright 2022 The IREE Authors
```

Reply:
Done.